src/messages/MMDSFindIno.h

```diff
@@ -22,7 +22,7 @@ struct MMDSFindIno : public Message {
   ceph_tid_t tid;
   inodeno_t ino;
 
```

Comment:
I would initialize them here (guaranteed), which can be overridden by the initializer list if needed.